lib/crawler_web/templates/home/index.html.heex

```diff
@@ -1,7 +1,48 @@
-<div class="mt-3">
+<nav class="mt-3" aria-label="breadcrumb">
```

Comment:
the form inside a `nav` seems a bit weird in term of HTML semantic @vnntsu I prefer the div instead FYI ;)

Reply:
Fix in [e30ade7](https://github.com/vnntsu/elixir-ic/pull/36/commits/e30ade7853ad15da4e2ca9ca7f92f5a63f2ee6df)